draft-ietf-quic-multipath.md

```diff
@@ -1337,12 +1325,15 @@ If a path is not actively used for a while, it might not be usable anymore,
 e.g. due to middlebox timeouts. To avoid such path breakage, endpoints
 can send ack-eliciting packets such as packets containing PING frames
 ({{Section 19.2 of QUIC-TRANSPORT}}) on that path to keep it alive.
-As discussed in
-{{Section 10.1.2 of QUIC-TRANSPORT}}, the keep-alive interval depends
-on the timeout in the middlebox.
-
-If a path was not actively used for a while, an endpoint can
-probe it before switching to active use if there are still other paths
+{{Section 5.3 of QUIC-TRANSPORT}} defines an optional keep alive process.
+This process can be applied to each path separately depending on application needs.
+Some applications may decide to not keep any not-actively used path alive,
+keep only one additonal path alive, or multiple paths, e.g. for more redunancy.
+As discussed in {{Section 10.1.2 of QUIC-TRANSPORT}}, the keep-alive interval
+needs to incorperate timeouts in middleboxes on the path.
+
+If a path was not actively used for a while and no keep alives have been sent,
+an endpoint can probe it before switching to active use if there are still other paths
 that are currently usable.
 
```

Comment:
The weird part of that paragraph is the conditional "if there are still other paths
that are currently usable." What we are saying is that and endpoint that want to use an idle path should probe it first, to verify that it is still usable. Of course if there is no other path available, the probing is a bit pointless, because if that last idle path does not work the connection will just break. But we we don't really need to say that.

Reply:
Yes, but I don't see a real problem just saying it anyway and keeping the sentence as it is...